_meeting-notes/15-10-19.md

```diff
@@ -0,0 +1,123 @@
+Test Ground Weekly
+===
+
+###### tags: `IPFS` `Testing`
+
+:::info
+- **Host:** @daviddias
+- **Location:** https://protocol.zoom.us/j/299213319
+- **Date:** Oct 15, 2019 5:00 PM (UTC)
+- **Participants:**
+    - @daviddias
+    - @jimpick
+    - @momack2
+- **Agenda**
+  1. Sprint Update `20min`
+  2. Identify blockers / Answer Questions `20min`
+  3. This week's planning `20min`
+- **Reference:** 
+  - [Last week meeting minute](/s/template-meeting-note)
+  - [Trello Board](https://trello.com/b/O9z3ljaH/testground-ignition-%F0%9F%9A%80)
+  - [Testing Infra Design](https://github.com/ipfs/testground/blob/master/docs/SPEC.md)
+  - [Testing Inventory](https://github.com/ipfs/testground/blob/master/docs/test-inventory.md)
+  - [Testing Improvement Tracker](https://docs.google.com/spreadsheets/d/1xyqyGUF-oe3x9ln88YonVeOMWWdknik74lVgL_3dBY8/edit#gid=0)
+
+:::
+
+## :mega: Update (Done, Shipped, Made Progress)
+
+- ==@raulk==
+  - Shipped
+- ==@daviddias==
+  - Explored and tinkered a ton how to get go-ipfs to work as a library vs working through an IPTB wrapper, ended up:
+    - Blazing through the go-ipfs forest to get a working example unixfs.Get with go-ipfs as a Library https://github.com/ipfs/go-ipfs/pull/6695
+    - Improving smlbench plan (that uses go-ipfs through IPTB) https://github.com/ipfs/testground/pull/60 
+  - Iterated on the Test Narratives design https://github.com/ipfs/testground/pull/33
+  - Improved the DHT Test Plan docs and structure https://github.com/ipfs/testground/pull/59
+  - Sync with Jim on TestGround plan and priorities https://github.com/ipfs/testground/pull/63
+  - Restructured how we run meetings and how we follow up to action items  https://github.com/ipfs/testground#team
+- ==@jimpick==
+  - (was sick much of last week)
+  - Sync with David on TestGround plan and priorities https://github.com/ipfs/testground/pull/63
+  - Elasticsearch ... using "Elasticsearch" service via AWS Marketplace
+    - Fairly generic Filebeat setup (JSON, Container input). Was missing data, fix is to increase polling frequency to 1s or tell testground to retain containers. https://gist.github.com/jimpick/fb3e5c841ae5d8c2436a5f8654bb0cf2
```

Comment:
@jimpick can you create a file on /docs for infrastructure notes and link things like this example?